rmgpy/rmg/reactors.py

```diff
@@ -463,10 +463,23 @@ def generate_reactor(self, phase_system):
         if self.evap_cond_conditions:
             kLA_kH_evap_cond = rms.kLAkHCondensationEvaporationWithReservoir(domain,self.evap_cond_conditions)
             interfaces.append(kLA_kH_evap_cond)
-            
+
         react = rms.Reactor(domain, y0, (0.0, self.tf), interfaces, p=p)
         return react, domain, interfaces, p
 
+class ConstantTPIdealGasReactor(Reactor):
+    def __init__(self, core_phase_system, edge_phase_system, initial_conditions, terminations, constant_species=[]):
+        super().__init__(core_phase_system, edge_phase_system, initial_conditions, terminations, constant_species=[])
```

Comment:
Can you add a line after this function? I also realize that having a mutable default argument isn’t the best, even though we are not mutating it later in the function.

Reply:
Fixed